src/cartesi-machine.lua

```diff
@@ -257,8 +257,80 @@ where options are:
   --auto-reset-uarch-state
     automatically reset state after miroarchitecture halt
 
-  -i or --htif-console-getchar
-    run in interactive mode.
+  --htif-console-getchar
+    run in interactive mode
+
+  --virtio-console
+    add a VirtIO console device,
+    this console is more responsive than HTIF console and has support for terminal size
+
+  -i
+    run in an interactive mode using a VirtIO console device
+    same as --htif-console-getchar --virtio-console
+
+  --share-dir=<directory>
+    Add a VirtIO Plan 9 filesystem device sharing a host directory with the guest.
+
+    This argument can be specified multiple times, each filesystem device will have
+    a tag (e.g. vfs0, vfs1, ...) that keeps incrementing to be used with the mount command.
+
+    The filesystem can be mounted in the guest with the following command:
+      mount -t 9p vfs0 /mnt
+
+  --net-dev=user
+    Add a VirtIO network device using host user-space networking,
+    this effectively allows to use host network from inside the machine with ease.
+
+    This is a user-mode network,
+    meaning you don't need root privilege or any configuration in the host to use it,
+    in most case it should work out of the box.
+
+    While being of use, this mode has the following limitations:
+      - There is an additional an emulation layer of the TCP/IP stack, so there is an overhead.
```

Comment:
Can you fix line breaks to make sure they work in 80 columns? I think there are older lines that are too long as well.